perun-base/src/main/resources/perun-roles.yml

```diff
@@ -5985,6 +5993,16 @@ perun_policies:
     include_policies:
       - default_policy
 
+  getApplicationsForGroup_Vo_List<String>_String_String_policy:
```

Comment:
First parameter of the method is a Group not a VO. Therefore, the name should be:
`getApplicationsForGroup_Group_List<String>_String_String_policy`

Reply:
Fixed.